site/src/api/errors.ts

```diff
@@ -110,15 +110,15 @@ export const getValidationErrorMessage = (error: unknown): string => {
   return validationErrors.map((error) => error.detail).join("\n");
 };
 
-export const getErrorDetail = (error: unknown): string | undefined | null => {
+export const getErrorDetail = (error: unknown): string | null => {
```

Comment:
Didn't see any situation where it made sense to make a user of the function have to split hairs between null and undefined

Reply:
I think we should return `string` or `undefined` instead of `null` if it is not a problem since we extensively use `undefined` in the codebase.